uxarray/core/dataset.py

```diff
@@ -445,24 +445,38 @@ def to_array(self) -> UxDataArray:
 
     def to_xarray(self, grid_format: str = "UGRID") -> xr.Dataset:
         """
-        Converts a ``ux.UXDataset`` to a ``xr.Dataset``.
-
-        Parameters
-        ----------
-        grid_format : str, default="UGRID"
-            The format in which to convert the grid. Supported values are "UGRID" and "HEALPix". The dimensions will
-            match the selected grid format.
-
-        Returns
-        -------
-        xr.Dataset
-            The ``ux.UXDataset`` represented as a ``xr.Dataset``
```

Comment:
These changes should not be made here. The purpose to `to_xarray()` for `UxDataset` to provide a method to fall back down to Xarray, which in this case doesn't have anything to do with the Grid information.